e2e/cypress/integration/files_and_attachments/image_link_preview_spec.js

```diff
@@ -89,4 +91,31 @@ describe('Image Link Preview', () => {
         // # All image previews expand back open
         cy.findAllByLabelText('file thumbnail').should('be.visible').and('have.length', 4);
     });
+
+    // https://automation-test-cases.vercel.app/test/MM-T1448
+    it.only('MM-T1448 Exit full screen mode when viewing a YouTube video', () => {
+        // # Post a youtube link
+        cy.postMessage('https://youtu.be/cIiDY4WA0oo');
+
+        cy.getLastPostId().then((youtubePost) => {
+            // # Move to the last posted youtube video
+            cy.get(`#post_${youtubePost}`).within(() => {
+                // # Wait a little until image preview of youtube video is downloaded
+                cy.wait(TIMEOUTS.FIVE_SEC);
+
+                // # Play the youtube video by clicking on image preview on video
+                cy.findAllByAltText('youtube video thumbnail').should('exist').and('be.visible').click({force: true});
+
+                // # Let the video play for few seconds
+                cy.wait(TIMEOUTS.THREE_SEC);
+
+                // # Pause the video so youtube controls are visible again
+                cy.getIframe().find('video').should('exist').click({force: true});
+
+                cy.getIframe().find('video').should('exist').dblclick({force: true});
+
+                cy.getIframe().type('f');
```

Comment:
There are no assertions added after typing `f`. @saturninoabril, I believe you were already having some conversation regarding this. Hence can you please advice further.